app/src/main/java/com/android/ubclaunchpad/driver/session/SessionInfoActivity.java

```diff
@@ -185,4 +214,63 @@ public void onCancelled(DatabaseError databaseError) {
                     }
                 });
     }
+
+    /**
+     * Retrieve all drivers and passengers from the current session
+     */
+    private void setDriverPassengers() {
+
+        final List<User> users = new ArrayList<>();
+
+        DatabaseReference usersReference = FirebaseUtils.getDatabase().child("Users");
+        // Getting reference to the users in Firebase
+        usersReference.addListenerForSingleValueEvent(new ValueEventListener() {
+            @Override
+            public void onDataChange(final DataSnapshot dataSnapshotUsers) {
+                // Getting reference to the drivers in current session
+                session.addListenerForSingleValueEvent(new ValueEventListener() {
+                    @Override
+                    public void onDataChange(DataSnapshot dataSnapshot) {
+
+                        DataSnapshot dataSnapshotDrivers = dataSnapshot.child(StringUtils.FirebaseSessionDriverEndpoint);
+
+
+                        for (DataSnapshot driver : dataSnapshotDrivers.getChildren()) {
+                            String driverUid = driver.getValue(String.class);
+
+                            User uDriver = dataSnapshotUsers.child(driverUid).getValue(User.class);
+                            uDriver.setIsDriver(true);
+                            uDriver.setUserUid(driverUid);
+                            users.add(uDriver);
+                        }
+                        DataSnapshot dataSnapshotPassengers = dataSnapshot.child(StringUtils.FirebaseSessionPassengerEndpoint);
+                        for (DataSnapshot passenger : dataSnapshotPassengers.getChildren()) {
+                            String passengerUid = passenger.getValue(String.class);
+                            User uPassenger = dataSnapshotUsers.child(passengerUid).getValue(User.class);
+                            uPassenger.setUserUid(passengerUid);
+                            users.add(uPassenger);
+
+                        }
+
+                        String locStr = dataSnapshot.child("location").getValue(String.class);
```

Comment:
can you make this a string constant too? other than that, feel free to merge the branch